portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -63,6 +63,8 @@ function onLoad() {
   fetchSubmittedLocations().then((locationData) => {
     populateDropdown(locationData, 'user-submitted-dropdown-menu');
   });
+
+  loadChartData();
```

Comment:
is there a purpose for this function to be called here? as of now it just looks like it's droppnig the chart data and not being used by anything

first guess would be to avoid the lint check errors?

Reply:
I was working a bit ahead of Bria, so she will use the result of this function to populate the charts feature. And this would happen onLoad.